### PR TITLE
*: Add default receivers

### PR DESCRIPTION
**Pull request by brancz**
This patch adds a few out of the box receivers that only need their
notification provider configuration filled in, instead of figuring out
all the wiring for critical alerts for example.

@s-urbaniak @paulfantom

### CODE REVIEW

**Comment by s-urbaniak** [comment]
lgtm

File: jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet [hunk 1]
```diff
@@ -40,19 +40,31 @@ local k = import 'ksonnet/ksonnet.beta.4/k.libsonnet';
...
+            },
           ],
         },
         receivers: [
           {
-            name: 'null',
+            name: 'Default',
+          },
+          {
+            name: 'WatchDog',
```
Comment (paulfantom) [inline jsonnet/kube-prometheus/alertmanager/alertmanager.libsonnet]:
```suggestion
            name: 'Watchdog',
```

File: jsonnetfile.lock.json [hunk 2]
```diff
@@ -1,67 +1,71 @@
 {
```
Comment (LiliC) [inline jsonnetfile.lock.json]: Do we want this in 4.4 version, if yes can you please update the deps in a separate commit so its easier to backport it to the release branch?

Comment (brancz) [reply jsonnetfile.lock.json]: The release-0.4 branch does not have this problem.

**Pull request merged**